DataConnectors/ESETPROTECT/Connector_Syslog_ESETPROTECT.json

```diff
@@ -0,0 +1,154 @@
+{
+    "id": "ESETPROTECT",
+    "title": "ESET PROTECT",
+    "publisher": "ESET Netherlands",
+    "descriptionMarkdown": "Syslog is an event logging protocol that is common to Linux. Applications will send messages that may be stored on the local machine or delivered to a Syslog collector. When the Agent for Linux is installed, it configures the local Syslog daemon to forward messages to the agent. The agent then sends the message to the workspace.\n\n[Learn more >](https://aka.ms/sysLogInfo)",
```

Comment:
descriptionMarkdown should meaningful explanation of what data the data connector enables bringing in, in what format and pointer to appliance documentation as applies.